### PR TITLE
seed-dashboard

### PR DESCRIPTION
**Pull request by kpslp**
for seed dashboard

### CODE REVIEW

**Review by eu-tee — APPROVED** [review]
Thank you huda

File: seed-dashboard/seed-dashboard-faq.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 >**Tip:** Click the triangle to view the answer.
 
 <details>
-<summary style="font-size:18px">How does DEEP dashboard work? </summary>
+<summary style="font-size:18px">How does SEED dashboard work? </summary>
 
-DEEP uses data from TechPass, Cloudflare, Microsoft Defender and Intune, to identify and prevent insecure devices from accessing protected resources. Additionally, DEEP measures device compliance against CIS-based benchmarks and automatically corrects non-compliant configurations. 
+SEED dashboard uses data from TechPass, Cloudflare, Microsoft Defender and Intune, to identify and prevent insecure devices from accessing protected resources. Additionally, DEEP measures device compliance against CIS-based benchmarks and automatically corrects non-compliant configurations.
```
Comment (cricketer94) [inline seed-dashboard/seed-dashboard-faq.md:7]: Hmmm this is something that technically is done by the DEEP subsystem (I believe we still have mentions of DEEP in the SEED primer right?). Can we change it to the following?

"SEED Dashboard pulls data from the respective components of SEED (i.e. TechPass, Cloudflare, Microsoft Defender and Intune) and uses it to give an overview of the security posture of the device. Additionally, users can use the dashboard to find out how they can remediate their security issues and regain access to SEED-protected applications"

File: seed-dashboard/seed-dashboard-faq.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 <details>
-<summary style="font-size:18px">How does DEEP dashboard work? </summary>
+<summary style="font-size:18px">How does SEED dashboard work? </summary>
 
-DEEP uses data from TechPass, Cloudflare, Microsoft Defender and Intune, to identify and prevent insecure devices from accessing protected resources. Additionally, DEEP measures device compliance against CIS-based benchmarks and automatically corrects non-compliant configurations. 
+SEED dashboard uses data from TechPass, Cloudflare, Microsoft Defender and Intune, to identify and prevent insecure devices from accessing protected resources. Additionally, DEEP measures device compliance against CIS-based benchmarks and automatically corrects non-compliant configurations. 
 </details>
 <br>
 <details>
 <summary style="font-size:18px">How do I report a potential security issue?  </summary>
```
Comment (cricketer94) [inline seed-dashboard/seed-dashboard-faq.md:11]: Actually it should be:
Security issues should be reported via the "Report Vulnerability" button at the bottom

Any other issues reported via the "Report Issue" button.

File: seed-dashboard/seed-dashboard-faq.md [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
 
-Go to the icon on the top right-hand corner of the DEEP Dashboard and click **Report issue**. You will be redirected to the service request page. We will respond to your queries within three business days.  
+Go to the icon on the top right-hand corner of the SEED Dashboard and click **Report issue**. You will be redirected to the service request page. We will respond to your queries within three business days.  
 </details>
 <br>
 <details>
 <summary style="font-size:18px">I received a notification regarding suspicious and malicious behaviour on my SEED device, what should I do? </summary>
```
Comment (cricketer94) [inline seed-dashboard/seed-dashboard-faq.md:17]: Do we want to include "my access on SEED device is blocked?" as a scenario as well?

File: seed-dashboard/seed-dashboard-tour.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 # DEEP Dashboard tour
```
Comment (cricketer94) [inline seed-dashboard/seed-dashboard-tour.md:1]: The title should be changed to "SEED Dashboard tour"

**Review by cricketer94 — CHANGES_REQUESTED** [review]
There's some spots that haven't been changed from DEEP Dashboard to SEED Dashboard.

File: seed-dashboard/seed-dashboard-faq.md (hunk 2 above)
Comment (kpslp) [reply seed-dashboard/seed-dashboard-faq.md:11]: @cricketer94 which are you referring to? the how do i report a potential security issue?

**Pull request merged**